ddtrace/settings.py

```diff
@@ -102,18 +122,80 @@ def __init__(self, global_config, *args, **kwargs):
         :param kwargs:
         """
         super(IntegrationConfig, self).__init__(*args, **kwargs)
-
+        self.http = HttpConfig()
         self.global_config = global_config
 
     def __deepcopy__(self, memodict=None):
         new = IntegrationConfig(self.global_config, deepcopy(dict(self)))
+        new.http = deepcopy(self.http)
         return new
-
+      
```

Comment:
```suggestion
```

Reply:
I am the one who introduced this `flake8` issue, sorry :(

Reply:
ahaha forgiven 😄